assets/sass/components/user-input/googlesitekit-user-input-questions.scss

```
@@ -23,6 +23,15 @@
 		border-radius: 0 0 24px 24px;
```

We have a variable with this value for border-radius (`24px`) - `$br-md`
```suggestion
                border-radius: 0 0 $br-md $br-md;
```